src/games/strategy/triplea/ai/proAI/ProData.java

```diff
@@ -0,0 +1,43 @@
+package games.strategy.triplea.ai.proAI;
+
+import games.strategy.engine.data.GameData;
+import games.strategy.triplea.Properties;
+
+/**
+ * Pro AI data.
+ */
+public class ProData {
+
+  public static double winPercentage = 95;
+  public static double minWinPercentage = 75;
+  public static boolean areNeutralsPassableByAir = false;
+
+  private static GameData data;
+  private static ProAI proAI;
+
+  public static GameData getData() {
+    return data;
+  }
+
+  public static void setData(final GameData data) {
+    ProData.data = data;
+
+    // Set optimal and win percentages lower if not LL
+    if (!games.strategy.triplea.Properties.getLow_Luck(data)) {
+      winPercentage = 90;
+      minWinPercentage = 65;
+    }
+
+    // Set map properties
+    areNeutralsPassableByAir = (Properties.getNeutralFlyoverAllowed(data) && !Properties.getNeutralsImpassable(data));
```

Comment:
Exactly. That's essentially the point of the new ProData class. Hold high level relatively static info that translated from the game data owned by the engine. So properties and fields related to the data/player/map/etc that don't really change during an AI players phase are held here so all the AI classes can reference them.